scripts/ci-e2e.sh

```diff
@@ -42,6 +42,7 @@ source "${REPO_ROOT}/hack/util.sh"
 capz::util::ensure_azure_envs
 
 export LOCAL_ONLY=${LOCAL_ONLY:-"true"}
+export USE_EXISTING_IMAGE=${USE_EXISTING_IMAGE:-"false"}
```

Comment:
Slight preference for the positive version of this so it's opt-out instead of opt-in:

```suggestion
export BUILD_MANAGER_IMAGE=${BUILD_MANAGER_IMAGE:-"true"}
```